petpals/models.py

```diff
@@ -147,3 +147,19 @@ def recent_photo_paths(self):
 
     def __repr__(self):
         return f"Profile('{self.name}', '{self.species}', '{self.subspecies}', '{self.color}', '{self.tagline}', '{self.biography}')"
+
+
+class Messages(db.Model):
+    message_id = db.Column(db.Integer, primary_key=True)
+    conversation_id = db.Column(db.Integer, nullable=False)
+    sender_username = db.Column(
+        db.String(18), db.ForeignKey("user.username"), nullable=False
+    )
+    recipient_username = db.Column(
+        db.String(18), db.ForeignKey("user.username"), nullable=False
+    )
+    time_sent = db.Column(db.TIMESTAMP, nullable=False)
```

Comment:
should be `db.DateTime` and add `default=datetime.utcnow`

Reply:
Fixed.